services/web-app/components/code/code.module.scss

```diff
@@ -9,54 +9,41 @@
 @use '@carbon/react/scss/type' as type;
 @use '@carbon/react/scss/theme' as theme;
 @use '@carbon/react/scss/motion' as motion;
-/* stylelint-disable declaration-no-important */
-
-// scoped to div code to override global reset styles
-:global(div) code {
-  padding: 0 spacing.$spacing-03;
-  background-color: theme.$layer-accent;
-  border-radius: 4px;
-  font-size: 0.875em;
-}
```

Comment:
Missed this before, these styles are styling the base inline code in markdown, we aren't translating them to the inline Carbon code snippet for some reason. Probably should be vs. creating our own styles here, trying to remember why I did it this way. 

<img width="1152" alt="Screen Shot 2022-09-20 at 2 06 02 PM" src="https://user-images.githubusercontent.com/2753488/191343266-ddc55a19-7d92-4e13-b173-6040d65fdd2e.png">